From 72b062de6687502a9cd9498e70d2cd6c084527d0 Mon Sep 17 00:00:00 2001
From: Randy Barlow <randy@electronsweatshop.com>
Date: Fri, 10 Mar 2017 15:11:10 -0500
Subject: [PATCH] Hard code wait_for_sync() to look for x86_64.

---
 bodhi/server/consumers/masher.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bodhi/server/consumers/masher.py b/bodhi/server/consumers/masher.py
index 38e1eec3..978963fb 100644
--- a/bodhi/server/consumers/masher.py
+++ b/bodhi/server/consumers/masher.py
@@ -750,7 +750,7 @@ class MasherThread(threading.Thread):
             force=True,
         )
         mash_path = os.path.join(self.path, self.id)
-        arch = os.listdir(mash_path)[0]
+        arch = 'x86_64'
 
         release = self.release.id_prefix.lower().replace('-', '_')
         request = self.request.value
-- 
2.12.0

