From bab45bd7649cfc2d516f0b7a7653d0217021e201 Mon Sep 17 00:00:00 2001
From: Randy Barlow <randy@electronsweatshop.com>
Date: Tue, 12 Dec 2017 16:36:50 -0500
Subject: [PATCH] Correct the help text to remove spaces betwen builds.

fixes #2071

Signed-off-by: Randy Barlow <randy@electronsweatshop.com>
---
 bodhi/client/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bodhi/client/__init__.py b/bodhi/client/__init__.py
index 0e8cf66..110dc8d 100644
--- a/bodhi/client/__init__.py
+++ b/bodhi/client/__init__.py
@@ -183,7 +183,7 @@ def new(user, password, url, **kwargs):
     Create a new update.
 
     BUILDS: a comma separated list of Builds to be added to the update
-    (e.g. 0ad-0.0.21-4.fc26, 2ping-3.2.1-4.fc26)
+    (e.g. 0ad-0.0.21-4.fc26,2ping-3.2.1-4.fc26)
     """
 
     # Developer Docs
-- 
2.9.5

