From 0981e62941d827f292b190e4b0e29868a2189bcd Mon Sep 17 00:00:00 2001
From: Patrick Uiterwijk <puiterwijk@redhat.com>
Date: Thu, 1 Dec 2016 02:44:56 +0000
Subject: [PATCH] Skip builds that are not assigned to a release

Signed-off-by: Patrick Uiterwijk <puiterwijk@redhat.com>
---
 bodhi/server/consumers/signed.py | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/bodhi/server/consumers/signed.py b/bodhi/server/consumers/signed.py
index 4868662c..9284fd8d 100644
--- a/bodhi/server/consumers/signed.py
+++ b/bodhi/server/consumers/signed.py
@@ -64,6 +64,10 @@ class SignedHandler(fedmsg.consumers.FedmsgConsumer):
                 log.info("Build was not submitted, skipping")
                 return
 
+            if not build.release:
+                log.info('Build is not assigned to release, skipping')
+                return
+
             if build.release.pending_testing_tag != tag:
                 log.info("Tag is not pending_testing tag, skipping")
                 return
-- 
2.11.0

