From 39495964781f352e7b1867988d8fdcd3d7351d83 Mon Sep 17 00:00:00 2001
From: Randy Barlow <bowlofeggs@electronsweatshop.com>
Date: Fri, 29 Jul 2016 15:28:32 -0400
Subject: [PATCH] Set the version in the setup.py to the next release.

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index e2383ec..f10b97d 100644
--- a/setup.py
+++ b/setup.py
@@ -76,7 +76,7 @@ if sys.version_info[:3] < (2,5,0):
     requires.append('pysqlite')
 
 setup(name='bodhi',
-      version='2.0',
+      version='2.1.8',
       description='bodhi',
       long_description=README + '\n\n' +  CHANGES,
       classifiers=[
-- 
2.7.4

