--- bodhi/tools/client.py.orig	2015-08-18 14:33:48.568658447 -0600
+++ bodhi/tools/client.py	2015-08-18 14:33:50.952621524 -0600
@@ -228,6 +228,8 @@
                         if 'updates' in data:
                             for update in data['updates']:
                                 print(bodhi.update_str(update).encode("UTF-8"))
+                        elif 'title' in data:
+                            print(bodhi.update_str(data).encode("UTF-8"))
 
                 else:
                     builds = ",".join(args)
@@ -252,6 +254,8 @@
                     if 'updates' in data:
                         for update in data['updates']:
                             print(bodhi.update_str(update).encode("UTF-8"))
+                    elif 'title' in data:
+                        print(bodhi.update_str(data).encode("UTF-8"))
 
             elif opts.edit:
                 verify_args(args)
@@ -464,11 +468,12 @@
                     if data.has_key('tg_flash') and data['tg_flash']:
                         log.error(data['tg_flash'])
                         sys.exit(-1)
-                    if data['num_items'] > 1:
+                    total = data.get('num_items', data.get('total', 0))
+                    if total > 1:
                         for update in data['updates']:
                             print(bodhi.update_str(update, minimal=True).encode("UTF-8"))
                         log.info("%d updates found (%d shown)" % (
-                            data['num_items'], len(data['updates'])))
+                            total, len(data['updates'])))
                     else:
                         for update in data['updates']:
                             print(bodhi.update_str(update).encode("UTF-8"))
