Index: bodhi-0.7.5/setup.py
===================================================================
--- bodhi-0.7.5.orig/setup.py
+++ bodhi-0.7.5/setup.py
@@ -1,10 +1,9 @@
 #!/usr/bin/env python
 #
 # $Id: bodhi-temp-client-only.patch,v 1.1 2010/07/27 19:20:26 toshio Exp $
-__requires__='TurboGears[future]'
+#__requires__='TurboGears[future]'
 
 from setuptools import setup, find_packages
-from turbogears.finddata import find_package_data
 import os
 import re
 import glob
@@ -19,14 +18,13 @@ from distutils.dep_util import newer
 from setuptools.command.install import install as _install
 from setuptools.command.install_lib import install_lib as _install_lib
 
-from turbogears.finddata import find_package_data, standard_exclude, \
-        standard_exclude_directories
-
+#from turbogears.finddata import find_package_data, standard_exclude, \
+#        standard_exclude_directories
 
 excludeFiles = ['*.cfg.in']
-excludeFiles.extend(standard_exclude)
+#excludeFiles.extend(standard_exclude)
 excludeDataDirs = ['bodhi/static', 'comps']
-excludeDataDirs.extend(standard_exclude_directories)
+#excludeDataDirs.extend(standard_exclude_directories)
 
 poFiles = filter(os.path.isfile, glob.glob('po/*.po'))
 
@@ -138,11 +136,11 @@ data_files = [
 for langfile in filter(os.path.isfile, glob.glob('locale/*/*/*')):
     data_files.append((os.path.dirname(langfile), [langfile]))
 
-package_data = find_package_data(where='bodhi',
-                                 package='bodhi',
-                                 exclude=excludeFiles,
-                                 exclude_directories=excludeDataDirs)
-package_data['bodhi.config'].append('app.cfg')
+#package_data = find_package_data(where='bodhi',
+#                                 package='bodhi',
+#                                 exclude=excludeFiles,
+#                                 exclude_directories=excludeDataDirs)
+#package_data['bodhi.config'].append('app.cfg')
 
 
 from bodhi.release import NAME, VERSION, DESCRIPTION
@@ -162,8 +160,8 @@ setup(
         'install_data': InstallData,
     },
     install_requires = [
-        "TurboGears >= 1.0",
-        "TurboMail",
+        #"TurboGears >= 1.0",
+        #"TurboMail",
         "python_fedora",
         "markdown",
     ],
@@ -171,7 +169,7 @@ setup(
     data_files = data_files,
     zip_safe = False,
     packages = find_packages(),
-    package_data = package_data,
+    #package_data = package_data,
     keywords = [
         'turbogears.app',
     ],
